### PR TITLE
qa/suites: more whitelisting

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
this should silence some warnings in http://pulpito.ceph.com/kchai-2017-08-02_10:30:38-rados-wip-kefu-testing-distro-basic-smithi/

File: qa/tasks/thrashosds-health.yaml [hunk 1]
```diff
@@ -11,3 +11,5 @@ overrides:
       - \(OBJECT_
       - \(REQUEST_SLOW\)
       - \(TOO_FEW_PGS\)
+      - \(FS_DEGRADED\)
```
Comment (liewegas) [inline qa/tasks/thrashosds-health.yaml]: why would we get FS_DEGRADED from osds thrashing?

**Comment by tchaikov** [comment]
ceph task creats an fs by default if there are one or more mds. and i am
sure one of our upgrade test starts an mds. see the pulpito link for more
details.

File: qa/suites/rados/singleton/all/osd-backfill.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ tasks:
       - \(OSD_
       - \(PG_
       - \(OBJECT_
+      - \(POOL_APP_NOT_ENABLED\)
```
Comment (liewegas) [inline qa/suites/rados/singleton/all/osd-backfill.yaml]: fixed by https://github.com/ceph/ceph/pull/16794

**Comment by liewegas** [comment]
Even if an MDS is running for the tests, osd thrashing shouldn't trigger a degraded fs/mds...

**Comment by liewegas** [comment]
i think this https://github.com/ceph/ceph/pull/16820 covers it now

**Pull request closed without merging**